### PR TITLE
PLAT-1181-2 Implement DomMenuPopover

### PR DESCRIPTION
**Pull request by alexander-schmidt**
- Created `DomMenuPopover` which supersedes `AbstractEmfManagementActionPopover`.
- Popovers are now automatically "compact".
  - I.e. they have smaller paddings, compared to other popups.
  - Added `DomCssPseudoClasses.COMPACT`.
- Merged `dom-popover-style.css` into `dom-popup-style.css`.
- Fix `DomPopover`:
  - Fix CTOR setting a CSS class (this was bad because popups which extended `DomPopover` looked different than popovers which were created as reconfigured regular popups).
  - Fix nonsensical base class.

image

### CODE REVIEW

File: platform-dom/src/main/java/com/softicar/platform/dom/elements/popup/configuration/DomPopupDisplayMode.java [hunk 1]
```diff
@@ -14,47 +14,50 @@ public enum DomPopupDisplayMode {
 	 * A modal, dismissable, non-draggable {@link DomPopup} without header.
 	 * Provides a visual clue to its modality.
 	 */
-	DIALOG(false, false, DomPopupModalMode.MODAL_DISMISSABLE, DomPopupChildClosingMode.INTERACTIVE_ALL),
+	DIALOG(false, false, false, DomPopupModalMode.MODAL_DISMISSABLE, DomPopupChildClosingMode.INTERACTIVE_ALL),
```
Comment (oliver-richers) [inline platform-dom/src/main/java/com/softicar/platform/dom/elements/popup/configuration/DomPopupDisplayMode.java:17]: - this is getting out of hand, lets spinoff another PR to fix it :)
- I suggest it like this:
```java
public interface DomPopupDisplayModes {
IDomPopupDisplayMode DIALOG = new DomPopupDisplayMode()//
.setCompact(false)
.setMaxmized(false)
...
```

File: platform-dom/src/main/java/com/softicar/platform/dom/elements/time/day/DomDayPopover.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package com.softicar.platform.dom.elements.time.day;
 
 import com.softicar.platform.common.date.Day;
-import com.softicar.platform.dom.elements.popup.modal.DomPopover;
+import com.softicar.platform.dom.elements.popup.popover.action.DomPopover;
```
Comment (oliver-richers) [inline platform-dom/src/main/java/com/softicar/platform/dom/elements/time/day/DomDayPopover.java:4]: why is `DomPopover` in an `action` subpackage?

**Review by oliver-richers — APPROVED** [review]

File: platform-dom/src/main/java/com/softicar/platform/dom/elements/time/day/DomDayPopover.java (hunk 2 above)
Comment (alexander-schmidt) [reply platform-dom/src/main/java/com/softicar/platform/dom/elements/time/day/DomDayPopover.java]: oh, that should not have been this way. removed the `action` package.

File: platform-dom/src/main/java/com/softicar/platform/dom/elements/popup/configuration/DomPopupDisplayMode.java (hunk 1 above)
Comment (alexander-schmidt) [reply platform-dom/src/main/java/com/softicar/platform/dom/elements/popup/configuration/DomPopupDisplayMode.java:17]: Tought the same. Telescope doing telescope things.

**Review by oliver-richers — APPROVED** [review]

**Pull request merged**